egs/wsj/s5/steps/cleanup/internal/segment_ctm_edits_mild.py

```diff
@@ -0,0 +1,2065 @@
+#! /usr/bin/env python
+
+# Copyright 2016   Vimal Manohar
+#           2016   Johns Hopkins University (author: Daniel Povey)
+# Apache 2.0
+
+from __future__ import print_function
+import argparse
+import copy
+import logging
+import heapq
+import sys
+from collections import defaultdict
+
+"""
+This script reads 'ctm-edits' file format that is produced by align_ctm_ref.py
+and modified by modify_ctm_edits.py and taint_ctm_edits.py Its function is to
+produce a segmentation and text from the ctm-edits input.
+
+The ctm-edits file format that this script expects is as follows
+<file-id> <channel> <start-time> <duration> <conf> <hyp-word> <ref-word> <edit>
+['tainted']
+[note: file-id is really utterance-id at this point].
```

Comment:
Explain here and in the usage message how it differs from segment_ctm_edits.py.
You might want to add some kind of 'see also' in segment_ctm_edits.py to refer to this, also.